src/server/routes/auth_Routes/spotifyAuth/index.ts

```diff
@@ -0,0 +1,94 @@
+import express, { Request, Response, NextFunction } from "express";
+import axios from "axios";
+import qs from "qs";
+import env from "dotenv";
+import { blue } from "chalk";
+import { User } from "../../../database/models";
+env.config();
+
+export const router = express.Router();
+
+router.get(
+  "/loginStatus",
+  async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      // console.log(req.currentUser)
+      // req.currentUser.accessToken = 'asdasdgdsfgadsfg'
+      await req.currentUser.save();
+      // console.log(req.currentUser)
+      if (req.currentUser.accessToken !== "not logged in") {
+        res.send(true);
```

Comment:
if there's a user, res.send the user! 
if there's no user, res.sendStatus(404)!